backend/core/src/shared/email/email.service.ts

```diff
@@ -131,6 +131,24 @@ export class EmailService {
     )
   }
 
+  public async sendlisting(listing: Listing, user: User) {
+    if (this.configService.get<string>("NODE_ENV") == "production") {
+      Logger.log(
+        `Preparing to send a listing email to ${user.email} from ${this.configService.get<string>(
+          "EMAIL_FROM_ADDRESS"
+        )}...`
+      )
+    }
+
+    await this.send(
+      user.email,
+      this.polyglot.t("New Listings"),
+      this.template("send-listing")({
```

Comment:
minor nit: maybe rename this template to be something like `new-listing` since we'll have multiple listing emails?